docs/sources/upgrading/_index.md

```diff
@@ -33,6 +33,8 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ## Main / Unreleased
 
+## 2.7.0
```

Comment:
Shouldn't this be 2.8.0?   (See line 144)

Reply:
good catch!